JSDemos/Demos/Toolbar/Adaptability/Vue/App.vue

```diff
@@ -437,6 +464,10 @@ export default {
   margin-bottom: 5px;
 }
 
+.dx-texteditor {
```

Comment:
I don't see this change in another frameworks. Is it necessary?

Reply:
Created task of fix vue-wrapper